docs/deployment.md

```diff
@@ -55,3 +55,32 @@ Folder | URL prefix
 
 To add more folders, follow [Cloudinary's instructions](https://cloudinary.com/documentation/fetch_remote_images#auto_upload_remote_resources).
 
+## Django Migrations: is my deploy safe?
+
+### Context
+
+[Preboot](https://devcenter.heroku.com/articles/preboot) is a feature from Heroku that prevents downtime when deploying: Heroku still serves traffic to old dynos while creating new dynos at the same time. For up to 3 minutes, two versions of the code are running and talking to the same DB: backward-incompatible changes to the DB result in internal server errors from the old code.
+
+### How to apply backward-incompatible migrations
+
+Always announce in `#mofo-engineering` when you deploy backward-incompatible changes.
+
+#### Rename or change a field type
+
+**Three deploys:**
```

Comment:
It feels like we probably want a section in the development.md docs as well, that explains the process for working on a task that requires multiple deploys: we want that to happen over multiple PRs, so something like "rename blah.moo to blah.flurp" should end up being a tracking issue for multiple PRs

Reply:
I don't think it's a good idea in the current state of those dev/deploy docs: they're super disorganized and splitting them made things even worst. I've moved my changes in a "workflow.md" doc but the whole thing needs a serious reorg/rewrite.

Reply:
I down for a serious rewrite, but it'd be good to at least have a mention in the development text about that fact that if you're touching models/fields, please make sure to have read this part of the deployment docs, too.